testdata/script/literals.txtar

```diff
@@ -66,6 +66,8 @@ import (
 	. "test/main/imp_func"
 	. "test/main/imp_var"
 	. "test/main/imp_type"
+	"test/main/imp_mult"
+	. "test/main/imp_mult"
```

Comment:
can you also include a case where we give it a custom name? that is, the package would end up imported three times.